### DISCUSSION TITLE
How to format text in vscode.notify?

### DISCUSSION THREAD

**Discussion by liuhq** [vscode-neovim/vscode-neovim#2203] [Q&A] [answered]
for example, '\n', i tried using it for line breaks, but it didn't work.

code 👇

![image](https://github.com/user-attachments/assets/88766571-f321-4c63-bb82-03798eaddcf6)

result 👇

![image](https://github.com/user-attachments/assets/c92f1f8e-1e23-4631-8a07-d33f19436975)

(note: "Add newline", "Add" is the content get from register)

**Comment by justinmk** [comment]
That's a vscode limitation. AFAIK, newlines aren't supported in `vscode.window.showInformationMessage`. Some limited markdown is supported, and clickable URLs are possible, I think.

**Reply by liuhq** [reply]
> That's a vscode limitation. AFAIK, newlines aren't supported in `vscode.window.showInformationMessage`. Some limited markdown is supported, and clickable URLs are possible, I think.

That's sad, I was going to use it to show the register contents when pressing \<leader\>".
Do you have any other ideas to implement it? 👇

![image](https://github.com/user-attachments/assets/3f288548-d3d8-42f1-9b5f-73dbe27e0aa4)

**Reply by justinmk** [reply, marked as answer]
Yes, just use Nvim's `:echomsg` . This will print in the vscode Output view, which is the entire point of the Output view. It makes no sense to use vscode notifications for showing large amounts of text, instead of the Output view.
